xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -60,9 +70,449 @@ constexpr int GLYPH_STRENGTH_BOLD = 24;
 constexpr int GLYPH_STRENGTH_LIGHT = -48;
 constexpr int TAB_SPACE_LENGTH = 4;
 
+constexpr std::chrono::hours JOB_RETRY_TIMEOUT{24};
+constexpr std::chrono::hours JOB_CANCEL_TIMEOUT{0xFFFFFFFF};
```

Comment:
Can this be set to `std::numeric_limits<std::chrono::hours>::max()`?

Reply:
Yes thats work, cool and thanks!